hack/bazel-test.sh

```diff
@@ -5,4 +5,6 @@ source hack/config.sh
 
 bazel test \
     --config=${ARCHITECTURE} \
-    --test_output=errors -- //staging/src/kubevirt.io/client-go/... //pkg/... //cmd/... //tests/framework/...
+    --test_output=errors -- //pkg/virt-handler/... //pkg/virt-launcher/...
+
+#    --test_output=errors -- //staging/src/kubevirt.io/client-go/... //pkg/... //cmd/... //tests/framework/...
```

Comment:
That seems like it was added by accident.

Reply:
> There is something wrong with the rebate.

@rmohr, I tried quite a few things but could not figure how to fix the rebase issue. Can you please give me some guide on this? I think I can do something like 'git reset --hard' to some commit and I can re-apply the changes manually from there.

Reply:
git fetch origin master
git rebase -i origin/master

That should first show you that you rebase a single commit, then you can resolve possible merge conflicts.

When done, force push on your remote branch.